examples/image.json

```diff
@@ -128,6 +128,10 @@
         {
             "name": "homepage",
             "value": "https://example.com"
+        },
+        {
+            "name": "appc.io/executor/systemd/support-notify",
```

Comment:
This made me think that it could be a good choice to use `appc.io/executor/systemd/` as prefix for all the systemd related annotation. I will change the docs accordingly.